src/main.ts

```diff
@@ -198,9 +198,9 @@ const createWindow = async () => {
 app.whenReady().then(async () => {
   await initSettings()
   appSettings = await getSettings()
-  nativeTheme.themeSource = appSettings.appearance.theme
+  nativeTheme.themeSource = appSettings.general.appearance.theme
```

Comment:
Changes in this file are intended for testing purposes and will be reverted before merging.